ddb-client/src/main/java/org/opensearch/remote/metadata/client/impl/DDBOpenSearchClient.java

```diff
@@ -626,8 +628,9 @@ private String simulateOpenSearchResponse(
         }
         StringBuilder sb = new StringBuilder("{");
```

Comment:
What do you think about using an Objectmapper here: `ObjectMapper mapper = new ObjectMapper();` ?

Using StringBuilder for JSON construction is error-prone.

Reply:
As long as it passes the test cases I'm fine with that!